cli/azd/extensions/registry.json

```diff
@@ -595,184 +510,6 @@
-              "checksum": {
-                "algorithm": "sha256",
-                "value": "d3524de84401cedecfe42ea8909ea0e4d5ff3363c36d8a6a89dfcf7a15580baa"
-              },
-              "entryPoint": "microsoft-azd-extensions-linux-amd64",
-              "url": "https://github.com/Azure/azure-dev/releases/download/azd-ext-microsoft-azd-extensions_0.7.0/microsoft-azd-extensions-linux-amd64.tar.gz"
-            },
-            "linux/arm64": {
-              "checksum": {
-                "algorithm": "sha256",
-                "value": "9f871c80875f591a188c8e8ad05bae01a9ceea3fdb6ee3c04abaaeb22868d9b3"
-              },
-              "entryPoint": "microsoft-azd-extensions-linux-arm64",
-              "url": "https://github.com/Azure/azure-dev/releases/download/azd-ext-microsoft-azd-extensions_0.7.0/microsoft-azd-extensions-linux-arm64.tar.gz"
-            },
-            "windows/amd64": {
-              "checksum": {
-                "algorithm": "sha256",
-                "value": "1a0c672841a9ed839c69655e467144bc3244169a00f2d70e2decdbe58e2d34fb"
-              },
-              "entryPoint": "microsoft-azd-extensions-windows-amd64.exe",
-              "url": "https://github.com/Azure/azure-dev/releases/download/azd-ext-microsoft-azd-extensions_0.7.0/microsoft-azd-extensions-windows-amd64.zip"
-            },
-            "windows/arm64": {
-              "checksum": {
-                "algorithm": "sha256",
-                "value": "38693fd9ecb02847195f2c23e3ea9c3aa12ab84192b64999cd0d3e0c299aeadb"
-              },
-              "entryPoint": "microsoft-azd-extensions-windows-arm64.exe",
-              "url": "https://github.com/Azure/azure-dev/releases/download/azd-ext-microsoft-azd-extensions_0.7.0/microsoft-azd-extensions-windows-arm64.zip"
-            }
-          }
-        },
-        {
-          "version": "0.7.1",
-          "capabilities": [
-            "custom-commands"
-          ],
-          "usage": "azd x \u003ccommand\u003e [options]",
-          "examples": [
-            {
-              "name": "init",
-              "description": "Initialize a new AZD extension project.",
-              "usage": "azd x init"
-            },
-            {
-              "name": "build",
-              "description": "Build the AZD extension project.",
-              "usage": "azd x build"
-            },
-            {
-              "name": "pack",
-              "description": "Package the AZD extension project into a distributable format and add to local registry.",
-              "usage": "azd x pack"
-            },
-            {
-              "name": "publish",
-              "description": "Publish the AZD extension project to an extension source.",
-              "usage": "azd x publish"
-            },
-            {
-              "name": "release",
-              "description": "Create an new release of the AZD extension project to a Github repository.",
-              "usage": "azd x release"
```

Comment:
This is not right.
Maybe this is because you had an old registry or something.  

Let's remove the changes to the registry (all this file).  And make one single PR in the future to update the registry.
We need to create a release-pipeline to produce the signed binaries in azure-dev and then update the registry from there